cernopendata/config.py

```diff
@@ -281,7 +281,7 @@
             default_order='asc',
             order=2,
         ),
-        'title': dict(fields=['title.keyword'],
+        'title': dict(fields=['title.exact'],
```

Comment:
Sorting is well fixed :+1: Special characters appear in the front, which is OK for now, but we can also see later whether we could classify them into the alphabet, e.g. `/Btau` amongst `B`s rather than in front of the search results.